Source/PeleLM_F.F90

```diff
@@ -165,6 +166,54 @@ integer function pphys_getckspecname(i, coded)
     pphys_getckspecname = str_len - 1
  
   end function pphys_getckspecname
+
+  subroutine parse_composition(compo_string, compo_vec)
+  
```

Comment:
Please provide in comments a nontrivial example of a valid compo_string, such as "O2:0.233 N2:0.767", and where it should be specified